### PR TITLE
Cleanup 20221109

### PR DESCRIPTION
**Pull request by fromanirh**
Followup found after more field testing of #466 and #468

### CODE REVIEW

File: test/e2e/serial/tests/workload_unschedulable.go [hunk 1]
```diff
@@ -233,7 +233,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 
 			By(fmt.Sprintf("checking deployment pods have been handled by the topology aware scheduler %q but failed to be scheduled on any node", serialconfig.Config.SchedulerName))
 			pods, err := podlist.ByDeployment(fxt.Client, *deployment)
-			Expect(err).NotTo(HaveOccurred(), "Unable to get pods from Deployment %q:  %v", deployment.Name, err)
+			Expect(err).ToNot(HaveOccurred(), "Unable to get pods from Deployment %q:  %v", deployment.Name, err)
```
Comment (shajmakh) [inline test/e2e/serial/tests/workload_unschedulable.go:236]: I guess this and similar change should be part of the second commit.

File: test/e2e/sched/install/install_test.go [hunk 2]
```diff
@@ -96,6 +96,7 @@ var _ = Describe("[Scheduler] install", func() {
 			By("Check secondary scheduler pod is scheduled on a control-plane node")
 			podList, err := podlist.ByDeployment(e2eclient.Client, *deployment)
 			Expect(err).NotTo(HaveOccurred())
+			Expect(podList).ToNot(BeEmpty(), "cannot find any pods for DS %s/%s", deployment.Namespace, deployment.Name)
```
Comment (shajmakh) [inline test/e2e/sched/install/install_test.go:99]: here and below , if i'm not mistaken it should  be DP (deployment) instead of DS (daemonset)

Comment (fromanirh) [reply test/e2e/sched/install/install_test.go:99]: right, thanks! will fix

**Comment by fromanirh** [comment]
> thank you for the cleanup! I hope this commit doesn't cause any conflicts in cases of backporting in the future. do we even think to backport the tests changes?

nope, we usually don't backport changes in the serial suite, because we want to keep the suite compatible all across the supported versions. Things can change only when we graduate the API to v1, though.

**Comment by fromanirh** [comment]
I guess I can drop the second commit though, let's narrow down the changes.

**Pull request merged**